app/data_models/app_models.py

```diff
@@ -19,28 +18,30 @@ def __init__(self, eq_session_id, user_id, expires_at, session_data):
         self.eq_session_id = eq_session_id
         self.user_id = user_id
         self.session_data = session_data
-        self.created_at = datetime.now(tz=tzutc())
-        self.updated_at = datetime.now(tz=tzutc())
-        self.expires_at = expires_at.replace(tzinfo=tzutc())
+        self.created_at = datetime.now(tz=timezone.utc)
+        self.updated_at = datetime.now(tz=timezone.utc)
+        self.expires_at = expires_at.replace(tzinfo=timezone.utc)
 
 
 class UsedJtiClaim:
     def __init__(self, jti_claim, expires_at):
         self.jti_claim = jti_claim
-        self.expires_at = expires_at.replace(tzinfo=tzutc())
+        self.expires_at = expires_at.replace(tzinfo=timezone.utc)
 
 
 # pylint: disable=no-self-use
 class Timestamp(fields.Field):
     def _serialize(self, value, attr, obj, **kwargs):
         if value:
             # Timezone aware datetime to timestamp
-            return int(value.replace(tzinfo=tzutc()).strftime("%s"))
+            return int(datetime.timestamp(value))
```

Comment:
How come the timezone was dropped?

Reply:
This one is reverted back to what it was, I had some flake8 issues and had to change it but now it's fixed.

Reply:
The usage of `tzutc()` should be dropped for `timezone.utc`.